baybe/priors/basic.py

```diff
@@ -85,3 +90,25 @@ def _validate_order(self, _: Any, b: float) -> None:  # noqa: DOC101, DOC103
                 f"For {self.__class__.__name__}, the upper bound `b` (provided: {b}) "
                 f"must be larger than the lower bound `a` (provided: {self.a})."
             )
+
+
+@define(frozen=True)
+class BetaPrior(Prior):
+    """A beta prior parameterized by alpha and beta."""
+
+    alpha: float = field(converter=float, validator=gt(0.0))
+    """Alpha concentration parameter. Controls mass accumulated toward zero."""
+
+    beta: float = field(converter=float, validator=gt(0.0))
+    """Beta concentration parameter. Controls mass accumulated toward one."""
+
+    def to_gpytorch(self, *args, **kwargs):  # noqa: D102
+        raise NotImplementedError(
+            f"'{self.__class__.__name__}' does not have a gpytorch analog."
+        )
+
+    def to_tensor(self) -> Tensor:
```

Comment:
is there any specific reason you need this as tensor? Isnt this even a misnomer? Istn what you want the torch distribution object? Why would you access alpha and beta like that and not just `prior_objecct.alpha`
So it seems to be this should be `to_torch` giving a `torch.Distribution`

And properly integrated all other prior should have this function (perhaps out of PR scope) but then at least the base class could have a `NotImplementedError`

Reply:
discussed in meeting